src/data/services/DiscoveryDataApiService.js

```diff
@@ -2,6 +2,7 @@ import apiClient from '../apiClient';
 
 class DiscoveryDataApiService {
   static discoveryBaseUrl = `${process.env.DISCOVERY_API_BASE_URL}/api/v1`;
+  static autocompleteBaseUrl = `${process.env.DISCOVERY_API_BASE_URL}/admin/course_metadata`;
```

Comment:
The autocomplete is really in the admin??

Reply:
🤷‍♂️ Yup.  The url was chosen back in the heady days of first publisher.